grr/server/grr_response_server/gui/ui/lib/api_translation/client.ts

```diff
@@ -14,6 +14,15 @@ function createKnowledgeBase(kb: ApiKnowledgeBase): KnowledgeBase {
   };
 }
 
+export function translateClientLabel(label: ApiClientLabel): ClientLabel {
```

Comment:
Please add a one-liner JSDoc to all exported symbols (functions, constants, ...).